client/src/components/ModelGraphViewerComponent/ModelGraphViewerCytoscapeComponent/ModelGraphViewerCytoscapeComponent.js

```diff
@@ -216,6 +211,24 @@ export class ModelGraphViewerCytoscapeComponent extends React.Component {
     });
   }
 
+  sessionLayout() {
+    const cy = this.graphControl;
+    const el = cy.nodes("*");
+    const currentLayoutPositions = sessionService.getCurrentModelsLayoutPositions(this.layout);
+    if (Object.keys(currentLayoutPositions).length === 0) {
+      for (let i = 0; i < el.length; i++) {
+        if (!currentLayoutPositions[el[i].data("id")]) {
```

Comment:
What's this check for? Isn't this branch of the if/else starting with an empty array of `currentLayoutPositions`?

Reply:
You are right I had a different  implementation for this where I needed it but now I don't.